io/src/vtk_lib_io.cpp

```diff
@@ -240,6 +240,11 @@ pcl::io::vtk2mesh (const vtkSmartPointer<vtkPolyData>& poly_data, pcl::PolygonMe
   mesh.cloud.width = mesh.cloud.height = 0;
   mesh.cloud.is_dense = true;
 
+  if (poly_data->GetPoints () == nullptr)
+  {
+    PCL_ERROR ("[pcl::io::vtk2mesh] Given vtkPolyData does not contain any points.\n");
```

Comment:
Maybe rephrase to include nullptr - since, "contain any points", could just mean it was initialized correctly, but with 0 points.